website/docs/addon_deadline_admin.md

```diff
@@ -75,6 +75,16 @@ The dependent publishing job's pool uses `ayon+settings://deadline/publish/Proce
 The logic for publishing job pool assignment applies to tiling jobs.
 :::
 
+## Publish Plugins
+
+### Process submitted job on farm
+
+`ProcessSubmittedJobOnFarm`
+
+#### List of family names to transfer to generated instances (AOVs for example)
+
+`families_transfer` - Because each AOV becomes its own instance when publishing on the farm, this attribute will ensure the AOV instances have these families if present on the rendering instance.
```

Comment:
Admittedly reading this I still have no clue when I should be using this. Can we provide some example use cases? What families? What would fail otherwise? Also, why is it optional if things break otherwise? Why **wouldn't** I want to use this?

Reply:
I'm honestly not 100 % sure on the original intent of this.

The use case I stumbled upon was needing to exclude render passes from Ftrack, where removing `ftrack` from this list worked. Should we include that?

Looks like it got introduced here https://github.com/ynput/OpenPype/commit/ed1308f5c1db4d64fab8a83780d6bb9ebdb76a7b#diff-dec46e41df53359bec451543fcf17f0d30d294337b78c7da6b59e2d549e41704 4 years ago.

@antirotor do you have any memory of this?

Reply:
I have to say this is brutally cryptic and I have a feeling it was solving some production situation at the time. The functionality is actually useful for complex workflows, but super confusing for simpler ones. ....eeerh. not sure here

Reply:
If I remember correctly (and I don't for sure)  that was introduced because when we were creating new instances for render layers (+ AOVs / +light groups) we needed a way to transfer just some families to the new one - like `review`. So when `review` was set on the "original" instance, it was added to all created from it. I guess most of this functionality was later replaced - like the case of review. But there are bunch of others I don't know, like slates, etc.

Reply:
How about this for docs instead?

```
When publishing to Deadline the instance data is filtered. This setting ensure the families listed are transferred to through to Deadline.
```